spec/features/audit/report/csv_export_spec.rb

```diff
@@ -13,99 +7,140 @@ def content_disposition
+  end
 
-      expect(page).to have_content("Title,URL")
-      expect(page).to have_content("Example 1,https://gov.uk/example1")
-    end
+  scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
+    given_i_am_an_auditor_belonging_to_an_organisation
+    and_there_are_two_unassigned_content_items
+    when_i_filter_the_content_page_for_content_items_with_no_one_assigned_to_them
+    then_i_see_the_two_unassigned_content_items
+    when_i_navigate_to_audits_report_page
+    then_i_do_not_see_the_two_unassigned_content_items
+    and_i_see_the_one_content_item_assigned_to_me
+  end
 
-    scenario "Applying the filters to the export" do
-      visit audits_report_path
-      select "Anyone", from: "allocated_to"
+  def given_i_am_an_auditor_belonging_to_an_organisation
+    user = create(:user)
+    hmrc = create(
+      :content_item,
+      title: "HMRC",
+      document_type: "organisation",
+      allocated_to: user
+    )
+    example1 = create(:content_item,
+                      title: "Example 1",
+                      base_path: "/example1",
+                      allocated_to: user)
+    create(:link,
+           source_content_id: example1.content_id,
+           target_content_id: hmrc.content_id,
+           link_type: "primary_publishing_organisation")
+    create(:audit, content_item: example1)
+  end
 
-      select "HMRC", from: "Organisations"
-      click_on "Apply filters"
+  def then_i_receive_only_hmrc_related_content_items_in_the_report
+    expect(@audit_report).to have_content("Title,URL")
+    expect(@audit_report).to have_content("Example 1,https://gov.uk/example1")
+    expect(@audit_report).to have_no_content("Example 2,https://gov.uk/example2")
+  end
 
-      click_link "Export filtered audit to CSV"
-      expect(page).to have_content("Example 1,https://gov.uk/example1")
-      expect(page).to have_no_content("Example 2,https://gov.uk/example2")
-    end
+  def then_i_see_the_two_unassigned_content_items
+    expect(@audit_content_page).to have_no_content("Example 1")
+    expect(@audit_content_page).to have_content("Example 2")
+    expect(@audit_content_page).to have_content("Example 3")
+  end
 
-    scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
-      visit audits_path
-      select "Anyone", from: "allocated_to"
+  def when_i_navigate_to_audits_report_page
+    @audit_content_page.audits_progress_tab.click
+  end
 
-      choose "Audited"
+  def then_i_do_not_see_the_two_unassigned_content_items
+    expect(@audit_report_page).to be_displayed
+    expect(@audit_report_page).to have_no_content('2 Content items')
+  end
 
-      click_on "Apply filters"
-      expect(page).to have_content("Example 1")
-      expect(page).to have_no_content("Example 2")
+  def and_i_see_the_one_content_item_assigned_to_me
+    expect(@audit_report_page).to have_content('1 Content items')
```

Comment:
Although this doesn't fall into the "not seeing" traps like the assertion above, it still falls into the trap of just asserting that you see one item, but not exactly caring what that item is. That's _not_ what your step is describing. In fact you care a lot about what that one item is (the one assigned to you) - so you should assert that that is the case.

Reply:
As discussed yesterday, this assertion is based on the report page which does not show individual content item details, just a summary of how many content items are in the report, how many have been audited etc.